odl/discr/discr_ops.py

```diff
@@ -430,11 +552,10 @@ def adjoint(self):
         """Return the adjoint operator.
 
         Assuming implicit zero padding, the adjoint operator is given by the
-        negative of the `Divergence` operator
+        negative of the `Divergence` operator.
 
-        Note that the ``space`` argument of the `Divergence`
-        operator is not the range but the domain of the `Gradient`
-        operator.
+        Note that the ``space`` argument of the `Divergence` operator is not
+        the range but the domain of the `Gradient` operator.
```

Comment:
Now I understood it when I read the code of `Divergence`. This will be easier to understand if you're more explicit, like "The `Divergence` is constructed from a ``space`` as a product space operator ``space^n --> space``, hence we need to provide the ``domain`` of this operator."